x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/pagination/pagination.spec.ts

```diff
@@ -5,25 +5,27 @@
  */
 
 import { logout } from '../../lib/logout';
-import { HOSTS_PAGE_TABS } from '../../lib/urls';
+import { HOSTS_PAGE_TAB_URLS } from '../../lib/urls';
 import {
+  AUTHENTICATIONS_TABLE,
   getDraggableField,
   getPageButtonSelector,
   NAVIGATION_AUTHENTICATIONS,
   NAVIGATION_UNCOMMON_PROCESSES,
   NUMBERED_PAGINATION,
   SUPER_DATE_PICKER_APPLY_BUTTON,
+  UNCOMMON_PROCCESSES_TABLE,
 } from '../../lib/pagination/selectors';
-import { DEFAULT_TIMEOUT, loginAndWaitForPage, waitForTableLoad } from '../../lib/util/helpers';
+import { DEFAULT_TIMEOUT, loginAndWaitForPage } from '../../lib/util/helpers';
 
 describe('Pagination', () => {
   afterEach(() => {
     return logout();
   });
 
   it('pagination updates results and page number', () => {
-    loginAndWaitForPage(HOSTS_PAGE_TABS.authentications);
-    waitForTableLoad();
+    loginAndWaitForPage(HOSTS_PAGE_TAB_URLS.authentications);
+    cy.get(AUTHENTICATIONS_TABLE, { timeout: DEFAULT_TIMEOUT });
```

Comment:
subjective / optional: As opposed to deleting the `waitForTableLoad()` helper function, how would you feel about instead adding a `dataTestSubj` parameter to it, so this line, and the other tests that follow, would in-turn read:

```
waitForTableLoad(AUTHENTICATIONS_TABLE);
```

Subjectively, when reading the tests, I find the `waitForTableLoad` abstraction above useful, because the code in the spec would be describing _what_ is happening (waiting for a table), as opposed to _how_ it's happening (selecting an element and waiting with the default timeout). What do you think?

Reply:
For smoke testing build candidates in cloud, we often spin up an instance and then run `audibeat` on our Macs, because it's fast and easy to get data into the cloud instance. When we do this, the Uncommon processes tab is nearly-instantly populated, but the Authentications tab is not.

Would you be willing to update this pagination test to use the Uncommon processes tab to simplify validating cloud BCs?

Reply:
sure to both points! going to finish up a different PR and come back to this towards EOW. thanks